### ISSUE TITLE
UI Not Loading

### ISSUE THREAD

**Issue by JBassett** [custom-components/hacs#34]
## Version of HACS
VERSION = "0.4.0"

## Describe the bug
Upgraded from 0.2.1 -> 0.4.0 and upon HA restart the Community Panel doesn't appear in left had drawer.

![hacs_missing](https://user-images.githubusercontent.com/1051414/58748412-56c77480-8446-11e9-932f-cdda95e133f7.png)

## Debug log

```text

2019-06-01 08:14:00 WARNING (MainThread) [homeassistant.loader] You are using a custom integration for hacs which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you do experience issues with Home Assistant.
2019-06-01 08:14:02 WARNING (MainThread) [homeassistant.loader] You are using a custom integration for personalcapital which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you do experience issues with Home Assistant.
2019-06-01 08:14:03 WARNING (MainThread) [homeassistant.loader] You are using a custom integration for radarr_upcoming_media which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you do experience issues with Home Assistant.
2019-06-01 08:14:03 WARNING (MainThread) [homeassistant.loader] You are using a custom integration for sonarr_upcoming_media which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you do experience issues with Home Assistant.
2019-06-01 08:14:05 WARNING (SyncWorker_37) [sleekxmppfs.basexmpp] fulljid property deprecated. Use boundjid.resource
2019-06-01 08:14:16 INFO (MainThread) [custom_components.hacs.hacs] Trying to load existing data.
2019-06-01 08:14:16 DEBUG (MainThread) [custom_components.hacs.storage] Reading from datastore /config/.storage/hacs.
2019-06-01 08:14:17 INFO (MainThread) [custom_components.hacs.storage] Loading custom-components/sensor.brewdog
2019-06-01 08:14:17 DEBUG (MainThread) [custom_components.hacs.storage] custom-components/sensor.brewdog - manifest.json is missing.
2019-06-01 08:14:17 INFO (MainThread) [custom_components.hacs.storage] Loading custom-components/switch.hadockermon
2019-06-01 08:14:18 INFO (MainThread) [custom_components.hacs.storage] Loading custom-components/hassbian_config
2019-06-01 08:14:18 ERROR (Thread-6) [pychromecast.socket_client] [192.168.1.32:8009] Failed to connect, retrying in 5.0s
2019-06-01 08:14:19 DEBUG (MainThread) [custom_components.hacs.storage] custom-components/hassbian_config - Repository structure does not meet the requirements
2019-06-01 08:14:19 INFO (MainThread) [custom_components.hacs.storage] Loading custom-components/sensor.weatheralerts
2019-06-01 08:14:19 DEBUG (MainThread) [custom_components.hacs.storage] custom-components/sensor.weatheralerts - manifest.json is missing.
2019-06-01 08:14:19 INFO (MainThread) [custom_components.hacs.storage] Loading custom-components/sensor.wienerlinien
2019-06-01 08:14:20 DEBUG (MainThread) [custom_components.hacs.storage] custom-components/sensor.wienerlinien - manifest.json is missing.
2019-06-01 08:14:20 INFO (MainThread) [custom_components.hacs.storage] Loading custom-components/binary_sensor.hadockermon
2019-06-01 08:14:20 DEBUG (MainThread) [custom_components.hacs.storage] custom-components/binary_sensor.hadockermon - manifest.json is missing.
2019-06-01 08:14:20 INFO (MainThread) [custom_components.hacs.storage] Loading custom-components/sensor.rpi_power
2019-06-01 08:14:21 INFO (MainThread) [custom_components.hacs.storage] Loading custom-components/sensor.untappd
2019-06-01 08:14:21 INFO (MainThread) [custom_components.hacs.storage] Loading custom-components/sensor.authenticated
2019-06-01 08:14:22 DEBUG (MainThread) [custom_components.hacs.storage] custom-components/sensor.authenticated - manifest.json is missing.
2019-06-01 08:14:22 INFO (MainThread) [custom_components.hacs.storage] Loading custom-components/sickchill
2019-06-01 08:14:22 DEBUG (MainThread) [custom_components.hacs.storage] custom-components/sickchill - manifest.json is missing.
2019-06-01 08:14:22 INFO (MainThread) [custom_components.hacs.storage] Loading custom-components/custom_updater
2019-06-01 08:14:23 DEBUG (MainThread) [custom_components.hacs.storage] custom-components/custom_updater - manifest.json is missing.
2019-06-01 08:14:23 INFO (MainThread) [custom_components.hacs.storage] Loading custom-components/usps_mail
2019-06-01 08:14:23 DEBUG (MainThread) [custom_components.hacs.storage] custom-components/usps_mail - manifest.json is missing.
2019-06-01 08:14:23 INFO (MainThread) [custom_components.hacs.storage] Loading custom-components/camera.unsplash
2019-06-01 08:14:24 DEBUG (MainThread) [custom_components.hacs.storage] custom-components/camera.unsplash - manifest.json is missing.
2019-06-01 08:14:24 INFO (MainThread) [custom_components.hacs.storage] Loading custom-components/camera.combined
2019-06-01 08:14:24 DEBUG (MainThread) [custom_components.hacs.storage] custom-components/camera.combined - manifest.json is missing.
2019-06-01 08:14:24 INFO (MainThread) [custom_components.hacs.storage] Loading custom-components/sensor.custom_aftership
2019-06-01 08:14:25 DEBUG (MainThread) [custom_components.hacs.storage] custom-components/sensor.custom_aftership - manifest.json is missing.
2019-06-01 08:14:25 INFO (MainThread) [custom_components.hacs.storage] Loading custom-components/feedparser
2019-06-01 08:14:25 INFO (MainThread) [custom_components.hacs.storage] Loading custom-components/camera.multisource
2019-06-01 08:14:26 DEBUG (MainThread) [custom_components.hacs.storage] custom-components/camera.multisource - manifest.json is missing.
2019-06-01 08:14:26 INFO (MainThread) [custom_components.hacs.storage] Loading custom-components/sensor.personalcapital
2019-06-01 08:14:26 INFO (MainThread) [custom_components.hacs.storage] Loading custom-components/sensor.trakt
2019-06-01 08:14:27 INFO (MainThread) [custom_components.hacs.storage] Loading custom-components/climate.e_thermostaat
2019-06-01 08:14:27 DEBUG (MainThread) [custom_components.hacs.storage] custom-components/climate.e_thermostaat - manifest.json is missing.
2019-06-01 08:14:27 INFO (MainThread) [custom_components.hacs.storage] Loading custom-components/sensor.geoip
2019-06-01 08:14:28 DEBUG (MainThread) [custom_components.hacs.storage] custom-components/sensor.geoip - manifest.json is missing.
2019-06-01 08:14:28 INFO (MainThread) [custom_components.hacs.storage] Loading custom-components/media_player.braviatv_psk
2019-06-01 08:14:28 INFO (MainThread) [custom_components.hacs.storage] Loading custom-components/sensor.sonarr_upcoming_media
2019-06-01 08:14:29 INFO (MainThread) [custom_components.hacs.storage] Loading custom-components/sensor.radarr_upcoming_media
2019-06-01 08:14:29 INFO (MainThread) [custom_components.hacs.storage] Loading custom-components/sensor.owlintuition
2019-06-01 08:14:30 INFO (MainThread) [custom_components.hacs.storage] Loading custom-components/sensor.unifigateway
2019-06-01 08:14:30 INFO (MainThread) [custom_components.hacs.storage] Loading custom-components/sensor.ssh
2019-06-01 08:14:31 INFO (MainThread) [custom_components.hacs.storage] Loading custom-components/sensor.plex_recently_added
2019-06-01 08:14:32 INFO (MainThread) [custom_components.hacs.storage] Loading custom-components/sensor.ctabustracker
2019-06-01 08:14:32 DEBUG (MainThread) [custom_components.hacs.storage] custom-components/sensor.ctabustracker - manifest.json is missing.
2019-06-01 08:14:32 INFO (MainThread) [custom_components.hacs.storage] Loading custom-components/sensor.kodi_recently_added
2019-06-01 08:14:33 INFO (MainThread) [custom_components.hacs.storage] Loading custom-components/sensor.youtube
2019-06-01 08:14:33 DEBUG (MainThread) [custom_components.hacs.storage] custom-components/sensor.youtube - manifest.json is missing.
2019-06-01 08:14:33 INFO (MainThread) [custom_components.hacs.storage] Loading custom-components/hacs
2019-06-01 08:14:34 INFO (MainThread) [custom_components.hacs.storage] Loading custom-components/config_check
2019-06-01 08:14:34 DEBUG (MainThread) [custom_components.hacs.storage] custom-components/config_check - manifest.json is missing.
2019-06-01 08:14:34 INFO (MainThread) [custom_components.hacs.storage] Loading custom-components/sensor.yandex_maps
2019-06-01 08:14:34 DEBUG (MainThread) [custom_components.hacs.storage] custom-components/sensor.yandex_maps - manifest.json is missing.
2019-06-01 08:14:34 INFO (MainThread) [custom_components.hacs.storage] Loading custom-components/climate.programmable_thermostat
2019-06-01 08:14:35 DEBUG (MainThread) [custom_components.hacs.storage] custom-components/climate.programmable_thermostat - manifest.json is missing.
2019-06-01 08:14:35 INFO (MainThread) [custom_components.hacs.storage] Loading custom-components/sensor.file_restore
2019-06-01 08:14:35 DEBUG (MainThread) [custom_components.hacs.storage] custom-components/sensor.file_restore - manifest.json is missing.
2019-06-01 08:14:35 INFO (MainThread) [custom_components.hacs.storage] Loading custom-components/complimentr
2019-06-01 08:14:36 DEBUG (MainThread) [custom_components.hacs.storage] custom-components/complimentr - manifest.json is missing.
2019-06-01 08:14:36 INFO (MainThread) [custom_components.hacs.storage] Loading custom-components/sensor.avanza_stock
2019-06-01 08:14:36 INFO (MainThread) [custom_components.hacs.storage] Loading custom-components/gpodder
2019-06-01 08:14:37 INFO (MainThread) [custom_components.hacs.storage] Loading custom-components/sensor.airthings_wave
2019-06-01 08:14:37 DEBUG (MainThread) [custom_components.hacs.storage] custom-components/sensor.airthings_wave - manifest.json is missing.
2019-06-01 08:14:37 INFO (MainThread) [custom_components.hacs.storage] Loading custom-components/breaking_changes
2019-06-01 08:14:38 INFO (MainThread) [custom_components.hacs.storage] Loading custom-components/grocy
2019-06-01 08:14:38 INFO (MainThread) [custom_components.hacs.storage] Loading custom-components/qbo
2019-06-01 08:14:39 INFO (MainThread) [custom_components.hacs.storage] Loading custom-components/google_keep
2019-06-01 08:14:39 DEBUG (MainThread) [custom_components.hacs.storage] custom-components/google_keep - This repository is empty.
2019-06-01 08:14:39 INFO (MainThread) [custom_components.hacs.storage] Loading custom-cards/text-element
2019-06-01 08:14:40 INFO (MainThread) [custom_components.hacs.storage] Loading custom-cards/circle-sensor-card
2019-06-01 08:14:40 INFO (MainThread) [custom_components.hacs.storage] Loading custom-cards/state-attribute-element
2019-06-01 08:14:41 INFO (MainThread) [custom_components.hacs.storage] Loading custom-cards/vertical-stack-in-card
2019-06-01 08:14:41 INFO (MainThread) [custom_components.hacs.storage] Loading custom-cards/state-element
2019-06-01 08:14:42 INFO (MainThread) [custom_components.hacs.storage] Loading custom-cards/canvas-gauge-card
2019-06-01 08:14:42 INFO (MainThread) [custom_components.hacs.storage] Loading custom-cards/beer-card
2019-06-01 08:14:43 INFO (MainThread) [custom_components.hacs.storage] Loading custom-cards/ext-weblink
2019-06-01 08:14:44 INFO (MainThread) [custom_components.hacs.storage] Loading custom-cards/aftership-card
2019-06-01 08:14:45 INFO (MainThread) [custom_components.hacs.storage] Loading custom-cards/username-element
2019-06-01 08:14:45 INFO (MainThread) [custom_components.hacs.storage] Loading custom-cards/list-card
2019-06-01 08:14:46 INFO (MainThread) [custom_components.hacs.storage] Loading custom-cards/button-card
2019-06-01 08:14:46 INFO (MainThread) [custom_components.hacs.storage] Loading custom-cards/rmv-card
2019-06-01 08:14:47 INFO (MainThread) [custom_components.hacs.storage] Loading custom-cards/upcoming-media-card
2019-06-01 08:14:47 INFO (MainThread) [custom_components.hacs.storage] Loading custom-cards/surveillance-card
2019-06-01 08:14:48 INFO (MainThread) [custom_components.hacs.storage] Loading custom-cards/light-entity-row
2019-06-01 08:14:48 INFO (MainThread) [custom_components.hacs.storage] Loading custom-cards/button-entity-row
2019-06-01 08:14:49 INFO (MainThread) [custom_components.hacs.storage] Loading custom-cards/flex-table-card
2019-06-01 08:14:50 INFO (MainThread) [custom_components.hacs.storage] Loading custom-cards/secondaryinfo-entity-row
2019-06-01 08:14:50 INFO (MainThread) [custom_components.hacs.storage] Loading custom-cards/bar-card
2019-06-01 08:14:51 INFO (MainThread) [custom_components.hacs.storage] Loading custom-cards/roku-card
2019-06-01 08:14:51 INFO (MainThread) [custom_components.hacs.storage] Loading custom-cards/pc-card
2019-06-01 08:14:52 INFO (MainThread) [custom_components.hacs.storage] Loading custom-cards/config-template-card
2019-06-01 08:14:52 INFO (MainThread) [custom_components.hacs.storage] Loading custom-cards/spotify-card
2019-06-01 08:14:53 DEBUG (MainThread) [custom_components.hacs.storage] custom-cards/spotify-card - No acceptable js files found
2019-06-01 08:14:53 INFO (MainThread) [custom_components.hacs.storage] Loading custom-cards/podcast-card
2019-06-01 08:14:54 INFO (MainThread) [custom_components.hacs.storage] Loading custom-cards/untappd-card
2019-06-01 08:14:55 INFO (MainThread) [custom_components.hacs.storage] Loading custom-cards/group-element
2019-06-01 08:14:55 INFO (MainThread) [custom_components.hacs.storage] Loading custom-cards/text-divider-row
2019-06-01 08:14:56 INFO (MainThread) [custom_components.hacs.storage] Loading custom-cards/cover-element
2019-06-01 08:14:56 INFO (MainThread) [custom_components.hacs.storage] Loading custom-cards/radial-menu
2019-06-01 08:14:57 INFO (MainThread) [custom_components.hacs.storage] Loading custom-cards/playback-card
2019-06-01 08:14:58 DEBUG (MainThread) [custom_components.hacs.storage] custom-cards/playback-card - No acceptable js files found
2019-06-01 08:14:58 INFO (MainThread) [custom_components.hacs.storage] Loading custom-cards/bignumber-card
2019-06-01 08:14:58 INFO (MainThread) [custom_components.hacs.storage] Loading custom-cards/entity-attributes-card
2019-06-01 08:14:59 INFO (MainThread) [custom_components.hacs.storage] Loading custom-cards/gauge-card
2019-06-01 08:14:59 INFO (MainThread) [custom_components.hacs.storage] Loading custom-cards/group-card
2019-06-01 08:15:00 INFO (MainThread) [custom_components.hacs.storage] Loading custom-cards/home-setter
2019-06-01 08:15:01 INFO (MainThread) [custom_components.hacs.storage] Loading custom-cards/monster-card
2019-06-01 08:15:02 INFO (MainThread) [custom_components.hacs.storage] Loading custom-cards/plan-coordinates
2019-06-01 08:15:03 INFO (MainThread) [custom_components.hacs.storage] Loading custom-cards/muuri-grid
2019-06-01 08:15:04 DEBUG (MainThread) [custom_components.hacs.storage] custom-cards/muuri-grid - No acceptable js files found
2019-06-01 08:15:04 INFO (MainThread) [custom_components.hacs.storage] Loading custom-cards/decluttering-card
2019-06-01 08:15:06 INFO (MainThread) [custom_components.hacs.storage] Saving data

```

**Comment by ludeeus** [comment]
HA Version?

**Comment by JBassett** [comment]
HA Version: 0.93.2
Running Docker Image

**Comment by ludeeus** [comment]
Use `0.4.1`, that fixes issues with loading in HA version < 0.94

**Comment by JBassett** [comment]
Fixed, thanks @ludeeus!
